config/locales/nl.yml

```diff
@@ -606,6 +608,7 @@ nl:
       public_long: Aan iedereen tonen, ook op openbare tijdlijnen
       unlisted: Minder openbaar
       unlisted_long: Aan iedereen tonen, maar niet op openbare tijdlijnen
+    title: '%{name}: "%{quote}"'
```

Comment:
Please move this line below [line 603](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L603)